testdata/Off/cube.off

```diff
@@ -12,9 +12,9 @@ OFF
 1 0 1
 1 1 0
 1 1 1
-4 0 1 3 2
-4 2 3 7 6
-4 4 6 7 5
-4 0 4 5 1
-4 1 5 7 3
-4 0 2 6 4
+4 0 1 3 2 255 0 0 #red
+4 2 3 7 6 0 255 0 #green
+4 4 6 7 5 0 0 255 #blue
+4 0 4 5 1 0 255 0
+4 1 5 7 3 0 0 255
+4 0 2 6 4 255 0 0
```

Comment:
Later on also try with random values between 0 and 255

Reply:
Resolved fb61590